### ISSUE TITLE
Introduce ready to use layouts

### ISSUE THREAD

**Issue by mkocansey** [mkocansey/bladewind#257]

**Comment by mkocansey** [comment]
Yes. Something similar. Not necessarily the exact same things but common layouts we use in most of our apps as developers.

**Comment by hetmann** [comment]
@mkocansey do you have a list of this layouts you have in mind?
